.lintstagedrc

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Could these be in package.json, same as Gutenberg does?

Reply:
👍